listen.js

```diff
@@ -0,0 +1,5 @@
+const app = require("./app")
```

Comment:
We don't technically need this listen.js file as supertest will attach it's own listener